scripts/deploy.sh

```diff
@@ -27,6 +27,12 @@ function run_ansible {
   openstack-ansible ${ANSIBLE_PARAMETERS} --forks ${FORKS} $@
 }
 
+apt-get install python-dev -y
+curl https://raw.githubusercontent.com/pypa/pip/7.1.2/contrib/get-pip.py -o /opt/get-pip.py
```

Comment:
This doesn't work (as per gate failures) - get-pip.py provides no way to override the pip version it bootstraps because it is hardcoded to always get the latest version. To do what you want here I think you need to let it do it's thing and then use the installed pip to downgrade pip:

```python get-pip.py && pip install -I pip==7.1.2```